app/models/user.rb

```diff
@@ -39,4 +42,8 @@ def self.agents_to_notify
         .where(notify: true)
   end
 
+  def self.create_with_omniauth(info)
+    User.create(name: info['name'])
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.